workflows/prefect_utils/slurm_flow.py

```diff
@@ -392,10 +352,22 @@ def compute_hash_of_input_file(
     return input_files_hash.hexdigest()
 
 
+@task(log_prints=True)
+def store_nextflow_trace(orchestrated_cluster_job: OrchestratedClusterJob):
+    job_description: OrchestratedClusterJob.SlurmJobSubmitDescription = (
```

Comment:
Should we have a flag to raise an error if no trace is found? This data is quite important, and I would like to avoid losing it